asr_deepspeech/src/speech_library/examples/speech2text.py

```diff
@@ -52,22 +52,28 @@ def main():
 
     print("Speech manager initialized in {} secs".format(end_time - start_time))
 
-    output_file = args.out
 
-    f = wave.open(input_file, "rb")
-    data = f.readframes(-1)
-    f.close()
+    #output_file = args.out
+
+    # since file is opened using "with", no need to close explicitly
+    with wave.open(input_file, "rb") as f:
+        try:
+            data = f.readframes(-1)
+        except:
+            if f.closed == False:
+                f.close()
+
 
     t3 = time.time()
     sm.push_data(data, finish_processing=True)
     r, r1 = sm.get_result()
     print(time.time() - t3)
 
-    if r1 and output_file:
-        with open(output_file, "w") as f:
-            f.write(r)
-    else:
-        print("transcribed text: {}".format(r))
+
+    if r1:
+        print("transcribed text: Please uncomment the 68th line")
```

Comment:
line number incorrect

Reply:
fixed in cabf1272c5240e39836103f5147554ca8ea86acb